### DISCUSSION TITLE
Fail threshold fails with warnings/weak warnings

### DISCUSSION THREAD

**Discussion by Tereo725** [JetBrains/Qodana#177] [Q&A] [answered]
Is it possible to change the severity at which Qodana fails? 

We're using this:
https://www.jetbrains.com/help/qodana/qodana-yaml.html#Set+a+fail+threshold

We would like to configure that errors are not allowed and the job fails, but warnings and weak warnings are OK and do not trigger the threshold. Is that possible?

**Comment by brichbash** [comment, marked as answer]
Hello,

Unfortunately, that is not possible yet. We have a feature request covering the use case [on our tracker](https://youtrack.jetbrains.com/issue/QD-5031/Add-an-option-to-set-an-allowed-number-of-problems-per-severity-for-failThreshold). If you have a spare minute, please vote for it to get notified about updates and let us know there is a demand for it.
